jsapp/js/account/plan.component.tsx

```diff
@@ -86,7 +86,7 @@ function planReducer(state: PlanState, action: DataUpdates) {
     case 'year':
       return {
         ...state,
-        intervalFilter: 'year',
+        intervalFilter: 'month',
```

Comment:
The interval toggle isn't working because this line was changed - 'month' is repeated twice.